versioned_docs/version-1.0/tutorial/01-first-steps.md

```diff
@@ -40,20 +45,70 @@ with deploying Cadence smart contracts and interacting with transaction and scri
 
 The Playground comes pre-loaded with contract and transaction templates
 that correspond to each of the tutorials in the docs site.
-To load the contracts from a specific tutorial, click the "Examples" link at the top of the Playground.
-This opens up a menu with each tutorial.
+To load the contracts for a specific tutorial, open the desired tutorial
+in the sidebar here and click the link in the callout box at the beginning of the tutorial.
+
+The callout boxes should look like this:
+
+<Callout type="success">
+  Open the starter code for this tutorial in the Flow Playground: <br />
+  <a
+    href="https://play.flow.com/"
+    target="_blank"
+  >
+    https://play.flow.com/
+  </a>
+</Callout>
+
+When you click on one of these links, the tutorial code will open in a new tab
+and the contracts, transactions, and scripts will be loaded 
+into the templates in the Playground for you to use.
+You will need to navigate between the editor and this tutorial to read instructions
+and make changes to your code.
+
+## What is a smart contract?
 
-When you click on one of these links, the tutorial will open in a new tab
-and the contracts, transactions, and scripts will be loaded into the templates in the Playground for you to use.
+In regular terms, a contract is an agreement between two parties for some exchange of information or assets.
+Normally, the terms of a contract are supervised and enforced by a trusted third party, such as a bank or a lawyer.
+
+A smart contract is a computer program stored in a network like a blockchain
+that verifies and executes the performance of a contract (like a lawyer does)
+without the need for any trusted third party anywhere in the process, because the code itself is trusted.
+
+Programs that run on blockchains are commonly referred to as smart contracts
+because they mediate important functionality (such as currency)
+without having to rely on a central authority (like a bank).
```

Comment:
```suggestion
Programs that run on blockchains are commonly referred to as smart contracts because they facilitate important functions, such as managing digital currency, without relying on a central authority like a bank.
```